docs/en/latest/plugins/traffic-split.md

```diff
@@ -141,13 +152,27 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
->Note: **1.** Use the `upstream_id` to bind the defined upstream, it can reuse upstream health detection, retry and other functions. **2.** Support the two configuration methods of `upstream` and `upstream_id` to be used together.
+:::tip
+
+Configure via `upstream_id` to reuse Upstream's health detection, retires, and other functions.
+
+:::
+
+:::note
 
-## Example
+You can use both `upstream` configuration and `upstream_id` configuration together.
 
-### Grayscale Release
+:::
 
-The `match` rule part is missing, and the traffic is split according to the `weight` value configured by the `weighted_upstreams` in the plugin. Divide `plugin's upstream` and `route's upstream` according to the traffic ratio of 3:2, of which 60% of the traffic reaches the upstream of the `1981` port in the plugin, and 40% of the traffic reaches the default `1980` port on the route Upstream.
+## Example usage
+
+The examples below shows different use cases for using the `traffic-split` Plugin.
+
+### Grayscale/Canary release
```

Comment:
Please use Canary Release instead

Reply:
Others LGTM

Reply:
So no need for mentioning Grayscale or Canary?

Reply:
Yes, only using the Canary release is enough

Reply:
Okay, done!